src/Microsoft.Framework.WebEncoders.Testing/CommonTestEncoder.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Framework.WebEncoders.Testing
```

Comment:
this looks consistent with `Microsoft.Framework.Logging.Testing` so I lean toward `Microsoft.Framework.WebEncoders.Testing` here.  anyone lean toward `Microsoft.Framework.WebEncoders`?  @Eilon?